src/types/userType.js

```diff
@@ -31,6 +31,7 @@ const userType = gql`
   }
 
   type Query {
+    me: User
```

Comment:
Just for the sake of best practices and conventional naming methods, can we change the `me` field to something like `currentUser` or something?

Reply:
Ok, I'll update this.